src/applications/personalization/dashboard/tests/components/cerner-widgets.unit.spec.js

```diff
@@ -12,22 +12,22 @@ describe('General Widget', () => {
   });
   it('renders the correct text, including the facility names', () => {
     expect(view.getByTestId('facilities').textContent).to.equal(
-      `For ${facilityNames[0]}: `,
+      `For ${facilityNames[0]}: Use My VA Health`,
```

Comment:
just FYI you could have also done:
```
to.contain(`For ${facilityNames[0]}:`)
```
or something like:
```
to.match(new RegExp(`^For ${facilityNames[0]}:`))
```

Reply:
Good to know. Thank you. Would you advise I change this before merging?